modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -488,7 +490,10 @@ private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue, S
                         Andes.getInstance().openLocalSubscription(subscriptionEvent);
                         alreadyAddedSubscriptions.add(subscriptionEvent);
                     }
+                } else {
+                    throw new AndesException("Binding already exists for the queue " + queue.getName());
```

Comment:
I think it will work in the cluster since we sync bind information in the cluster. But better to check.